### PR TITLE
Add parsers LvsAll, PvsAll, VgsAll and combiner LvmAll for getting LVM information for all accepted and rejected devices.

### PR DESCRIPTION
**Pull request by lonicerae**
- When executing the lvs, pvs, vgs commands on system, the collected data is
   generated following the 'filter' list in /etc/lvm/lvm.conf by default. But
   this make it impossible to find out the bad filters that blacklist normal
   disks. So that we need to add extra directives to collect LVM information
   for all devices.
 - Note that these parsers and combiner should not be used in ordinary plugins.
   They're used for checking the LVM filter issues.

### CODE REVIEW

**Comment by bfahr** [comment]
@lonicerae is your reference related to the Microsoft/vscode-python issue above correct?

**Comment by lonicerae** [comment]
@bfahr , deeply sorry for that!.. I was reading that bug and filled this PR link to that bug by mistake when I was writing the comment in this PR at the same time. I deleted the comment in the vscode-python at once but the github had already reference that comment.. Sorry for any confusion..

File: insights/combiners/tests/test_lvm.py [hunk 1]
```diff
@@ -452,3 +453,169 @@ def test_combiner_lvs(lvm_data):
...
+            has_vgsall = True
+            lvm_info = data.lvm_info
+            assert lvm_info.volume_groups is not None
+            assert len(list(lvm_info.volume_groups)) == 8
+            assert lvm_info.volume_groups['data7']['VG_UUID'] == 'r58iLs-sYla-lC4h-UURW-bZOG-P2Yd-lambpM'
+            assert lvm_info.volume_group_names == {
+                'data1', 'data2', 'data3', 'data4', 'data5', 'data6', 'data7',
+                'rhel_ceehadoop1'
+            }
+            if VgsAll in data.shared:
```
Comment (xiangce) [inline insights/combiners/tests/test_lvm.py]: This `if` can be removed and it's `else` can be shifted out as the `else` against to the `if` at line `477`.

File: insights/combiners/tests/test_lvm.py [hunk 2]
```diff
@@ -452,3 +453,169 @@ def test_combiner_lvs(lvm_data):
...
+                '/dev/sde1',
+                '/dev/sdf1',
+                '/dev/sdg1',
+                '/dev/sdh1'
+            }
+            pv_sdg1 = lvm_info.filter_physical_volumes('/dev/sdg1')
+            assert len(pv_sdg1) == 1
+            pv_sdg1_key, pv_sdg1_values = pv_sdg1.popitem()
+            assert '/dev/sdg1' in lvm_info.physical_volume_names
+            if PvsAll in data.shared:
```
Comment (xiangce) [inline insights/combiners/tests/test_lvm.py]: The same with above comment.

File: insights/combiners/tests/test_lvm.py [hunk 3]
```diff
@@ -452,3 +453,169 @@ def test_combiner_lvs(lvm_data):
...
+def test_combiner_lvsall(lvm_data_all):
+    has_lvsall = False
+    for data in lvm_data_all:
+        if LvsAll in data.shared:
+            has_lvsall = True
+            lvm_info = data.lvm_info
+            assert lvm_info.logical_volumes is not None
+            assert len(list(lvm_info.logical_volumes)) == 40
+            pool7 = lvm_info.logical_volumes[Lvm.LvVgName(LV='pool7', VG='data7')]
+            if LvsAll in data.shared:
```
Comment (xiangce) [inline insights/combiners/tests/test_lvm.py]: The same to above.

File: insights/combiners/lvm.py [hunk 4]
```diff
@@ -242,3 +243,38 @@ def filter_logical_volumes(self, lv_filter, vg_filter=None):
...
             return {k: v for k, v in self.logical_volumes.iteritems()
                     if lv_filter in k.LV and vg_filter in k.VG}
+
+
+@combiner(requires=[[LvsAll, PvsAll, VgsAll]])
+class LvmAll(Lvm):
+    """A Lvm like shared combiner for processing LVM information including all rejected
+        and accepted devices"""
+
+    LvVgName = namedtuple('LvVgName', ['LV', 'VG'])
```
Comment (xiangce) [inline insights/combiners/lvm.py]: This `LvVgname` can be removed from the sub-class, and use the Lvm.LvVgname directly.

**Comment by lonicerae** [comment]
Hey @xiangce, thanks for your suggestions, code updated, please help to review them again, thanks!

**Pull request merged**